### PR TITLE
Text and misc fixes

### LINKED ISSUE #1656: No word wrap in review text
Review text doesn't wrap words correctly.

OB v1.1.5

review2

### LINKED ISSUE #1665: Unnatural word break in chat
The chat breaks words in an unnatural fashion. See screenshot below. For example, `than` is `tha` + `n`.

screen shot 2016-06-07 at 09 29 37

### LINKED ISSUE #1670: Dispute summary does not provide buyer info.
When clicking on a dispute, the "summary" tab only provides the vendor and moderator info, but not the buyer info. If this is intentional, it is confusing...

### LINKED ISSUE #1672: Error in Categories field width
After updating to OB 1.1.6, categories field on item edit page got a line break when Brazilian Portuguese language is selected.

categories

### LINKED ISSUE #1673: Translated countries are out of alphabetical order
This behavior is seen if a non-english language is selected.

countries out of order

countries out of order2

### LINKED ISSUE #1674: "%" missing on moderator's fee
Started on v1.1.6

no

### LINKED ISSUE #1675: Chat message being considered a required field
Chat message is not a required field and should not be marked in red if nothing is typed.

It is now being marked in red even if user doesn't click on send message (i.e. clicks on chat text).

chat message input

### LINKED ISSUE #1676: Missing translation on date and time
date and time order

chat date and time

### LINKED ISSUE #1677: Incorrect text height
@ symbol is being cut at the bottom

line

### LINKED ISSUE #1678: Can't translate "Select Some Options" message on Ships To field
select some options

### LINKED ISSUE #1679: "Free" label on order window
"Free" label is not clear, and should be replaced by "Free shipping".

Another option could be adding a "Shipping Price" label on the first row of right column.

free ship

### LINKED ISSUE #1681: Consistent terminology in Dispute Panel
Change 'Seller' to 'Vendor' to make it consistent with the rest of the application.

image

PS: @jjeffryes I'm just gonna initially assign frontend issues to you, and you can hand it off to any else from there

### LINKED ISSUE #1684: Wrong title on Shipping & Returns tab
Instead of "Worldwide" should be "Domestic"
worldwide

### LINKED ISSUE #1685: Consistent terminology in moderator fee
It should be called either "_Dispute fee_" or "_Service fee_", since it is the same thing.

dispute fee

service fee

### LINKED ISSUE #1688: Pluralize image(s) if possible
The practice of making a word singular and plural using (s) doesn't exist in some other languages, it is much better to handle this with a polyglot plural phrase. 

If the number of images can be determined, change "We were unable to load your image(s)." into a phrase in polyglot with singular and plural forms.

If it's not possible to determine the number of images, change the phrase to something more generic, like "The image upload was not successful."

### LINKED ISSUE #1692: Translated currencies are out of alphabetical order
currency out of order

### PR DESCRIPTION
**Pull request by jjeffryes**
- Refactor showing NSFW content on user's own page if their settings are set to hide NSFW content
- Fix bug in blurring items that are not nsfw on nsfw pages.
- adds translations to chosen.js messages  …
- fixes text being cut off by the txt-unleaded class making the line-height too short
- translates dates set by moment.js 
- validates form when send button is pressed in the transaction modal chat
- fixes bug with boxUp tooltips not expanding to the size of their text
- adds % to moderator fees in the purchasing flow  …
- alphabetizes country names in settings after translating them  …
- fixes country select in address tab being too short
- sets flex columns to have maximum widths to compensate for the change in how flex-shrink is handled in Chrome
- alphabetizes the currencies in the settings view General tab based on the translated values
- changes break-all in CSS to break-word
- moves tooltips in edit item so they don't inherit transparency
- Changes the text on the Shipping tab of the item view so the Ships from is in the same column as the Ships to, to be less confusing
- Changes the text on the Shipping tab of the item view so international shipping is not shown if shipping is worldwide
- Changes the text on the Shipping tab of the item view so if there is only one shipping destination, it is shown inline, but multiple destinations are shown as a list
- Use Dispute fee in all cases for the moderator, instead of Service Fee when setting yourself as a moderator
- uses a smart count for the images error message
- add the buyer guid to the summary of cases in the transaction modal
- Closes #1656
- Closes #1665 
- Closes #1670
- Closes #1672
- Closes #1673
- Closes #1674 
- Closes #1675
- Closes #1676
- Closes #1677
- Closes #1678
- Closes #1679
- Closes #1681
- Closes #1684
- Closes #1685
- Closes #1688
- Closes #1692

### CODE REVIEW

File: js/languages/en-US.json [hunk 1]
```diff
@@ -551,7 +553,7 @@
     "pageUnavailable": "This page is currently unavailable.",
     "badHandle": "The handle you entered does not have a valid OpenBazaar ID.",
     "serverDown": "The server has been shut down",
-    "unableToLoadImages": "We were unable to load your image(s).",
+    "unableToLoadImages": "We were unable to load your image.  |||| We were unable to load your images.",
```
Comment (rmisio) [inline js/languages/en-US.json]: This doesn't look right..? You're calling it and passing in a `smart-count`, which doesn't appear to be used. Also, should those four vertical lines really be there (is there some weird polyglot plural/singular syntax I'm blissfully oblivious to?)

Comment (rmisio) [inline js/languages/en-US.json]: This doesn't look right..? You're calling it and passing in a `smart-count`, which doesn't appear to be used. Also, should those four vertical lines really be there (is there some weird polyglot plural/singular syntax I'm blissfully oblivious to?)

Comment (rmisio) [inline js/languages/en-US.json]: ah, ok. google is my friend. nevermind, I didn't know about this syntax. Good to know!

Now, will our translators know!?

Also, very minor, but there is an extra space in-between the first sentence and the vertical separators.

Comment (jjeffryes) [inline js/languages/en-US.json]: The smart count is passed in, and determines whether the first or second phrase should be used. It doesn't need to be in the actual phrase anywhere. The documentation isn't super clear on that. ;)

Hopefully the translators will know how to handle it, unfortunately we can't put comments into the language files. 

The extra spaces are a typo, I'll take them out.

File: js/templates/transactionModal.html [hunk 2]
```diff
@@ -638,13 +638,28 @@ <h3 class="txt-bright"><%= polyglot.t('CheckExchanges') %> </h3>
                     <strong><%- ob.vendor_offer.listing.item.title %></strong>
                   </a>
                 </div>
+                <% console.log(ob.transactionType)%>
```
Comment (rmisio) [inline js/templates/transactionModal.html]: do we want to commit these console.logs?

Comment (jjeffryes) [inline js/templates/transactionModal.html]: Nope.

Comment (jjeffryes) [inline js/templates/transactionModal.html]: Good catch.

**Pull request merged**